src/actions/auctionActions.js

```diff
@@ -2,28 +2,171 @@ import {
+      transactionHash: '0xac0c5460f5f9f9510ede073c2c62f47509f650d99efdac8733abdcb6550d6b93',
+      transactionIndex: 0,
+      transactionLogIndex: '0x1',
+      type: 'mined',
+      id: 'log_9aa9fa3f',
+      returnValues: {
+        '0': '17',
+        '1': '500000000000000000',
+        '2': '2500000000000000000',
+        auctionCounter: '17',
+        totalEtherConsumed: '500000000000000000',
+        totalNecBurned: '2500000000000000000',
+      },
+      event: 'AuctionClose',
+      signature: '0xa6cc937511bcbe4aa9f9693416797c7d255412e27bda9ef791a45903f7e97d4e',
+      raw: {
+        data:
+          '0x0000000000000000000000000000000000000000000000000de0b6b3a76400000000000000000000000000000000000000000000000000000000000000000000',
+        topics: [Array],
+      },
+    },
+    {
+      address: '0x8AaEEa652EBD90fB8D64A6cac09a0293CE62dD45',
+      blockHash: '0xdb165110dd56c1b1107854de915c239506a7ce3fc88a03c1dcae56379c7c80c7',
+      blockNumber: 15939734,
+      logIndex: 1,
+      removed: false,
+      transactionHash: '0xac0c5460f5f9f9510ede073c2c62f47509f650d99efdac8733abdcb6550d6b93',
+      transactionIndex: 0,
+      transactionLogIndex: '0x1',
+      type: 'mined',
+      id: 'log_9aa9fa3f',
+      returnValues: {
+        '0': '17',
+        '1': '1000000000000000000',
+        '2': '2000000000000000000',
+        auctionCounter: '17',
+        totalEtherConsumed: '1000000000000000000',
+        totalNecBurned: '2000000000000000000',
+      },
+      event: 'AuctionClose',
+      signature: '0xa6cc937511bcbe4aa9f9693416797c7d255412e27bda9ef791a45903f7e97d4e',
+      raw: {
+        data:
+          '0x0000000000000000000000000000000000000000000000000de0b6b3a76400000000000000000000000000000000000000000000000000000000000000000000',
+        topics: [Array],
+      },
+    },
+    {
+      address: '0x8AaEEa652EBD90fB8D64A6cac09a0293CE62dD45',
+      blockHash: '0xdb165110dd56c1b1107854de915c239506a7ce3fc88a03c1dcae56379c7c80c7',
+      blockNumber: 15939734,
+      logIndex: 1,
+      removed: false,
+      transactionHash: '0xac0c5460f5f9f9510ede073c2c62f47509f650d99efdac8733abdcb6550d6b93',
+      transactionIndex: 0,
+      transactionLogIndex: '0x1',
+      type: 'mined',
+      id: 'log_9aa9fa3f',
+      returnValues: {
+        '0': '18',
+        '1': '1000000000000000000',
+        '2': '1000000000000000000',
+        auctionCounter: '168',
+        totalEtherConsumed: '1000000000000000000',
+        totalNecBurned: '1000000000000000000',
+      },
+      event: 'AuctionClose',
+      signature: '0xa6cc937511bcbe4aa9f9693416797c7d255412e27bda9ef791a45903f7e97d4e',
+      raw: {
+        data:
+          '0x0000000000000000000000000000000000000000000000000de0b6b3a76400000000000000000000000000000000000000000000000000000000000000000000',
+        topics: [Array],
+      },
+    },
+  ];
+  pastEvents.map((event, index) => {
+    burnedNec.push({
+      name: `Point ${index}`,
+      pv: event.returnValues[2],
```

Comment:
FYI you can use `returnValues.totalNecBurned` instead (its the same value, but clearer to read in the code)